source/settings/command-line-options.txt

```diff
@@ -75,6 +70,13 @@ configuration file.
        Specify a passphrase to decrypt the :ref:`imported favorite
        connections <compass-import-connections-cli-encrypted>` file. 
 
+   * - .. option:: --showExampleConfig
+     - Provides an example |compass-short| configuration file. 
+
+   * - .. option:: --trustedConnectionString
+     - Suppresses warnings about disallowed connection string properties and allows 
+       automatic connection.
```

Comment:
This doc item should be more specific about what it does – i.e. _why_ some connection string properties are disallowed by default and what the ramifications are of allowing them/suppressing the relevant warnings.

Reply:
e.g., this text should explain that:

- This option applies if a connection string is passed on the command line
- Compass inspects the connection string passed in the command line, and verifies whether it contains parameters that the user may not want to have set and result in a security risk for the user (as a simple example, it could be `?tls=true` to disable TLS for the connection)
- This option should be set if and only if a _known_ trusted connection string is being passed to Compass, e.g. from a scrit

Reply:
Thank you, updated!